docs/src/main/asciidoc/security-oauth2.adoc

```diff
@@ -342,3 +342,74 @@ $ ./target/security-oauth2-quickstart-runner
 2019-03-28 14:31:37,315 INFO  [io.quarkus] (main) Quarkus 0.20.0 started in 0.006s. Listening on: http://[::]:8080
 2019-03-28 14:31:37,316 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy, resteasy-jsonb, security, security-oauth2]
 ----
+
+== Integration testing
+
+If you don't want to use a real OAuth2 authorization server for your integration tests, you can use the
+link:security-properties[Properties based security] extension for your test, or mock an authorization server using Wiremock.
+
+To mock an authentication server via Wiremock, you first need to add the Wiremock library inside your `pom.xml`.
+
+[source,xml]
+----
+<dependency>
+    <groupId>com.github.tomakehurst</groupId>
+    <artifactId>wiremock-jre8</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+Then you setup Wiremock to always return a valid response for any OAuth2 token send to the `/introspect` endpoint,
+you must adapt the response to fit your needs (at least the scope).
+
+[source,java]
+----
+private static WireMockServer wireMockServer = new WireMockServer();
```

Comment:
I am pretty sure that we want to do something like https://github.com/quarkusio/quarkus/blob/db21557225bab192c865212d7775c92175727769/docs/src/main/asciidoc/rest-client.adoc#L478 instead of this per pest configuration

Reply:
Bot approache are good for different reason.
If we prefere TestResource I'm OK to switch to it inside our integration test and update the guide.

Reply:
Not really, when you `%test.quarkus.oauth2.introspection-url=http://localhost:8080/introspect` it applies to all tests, so best make sure that you have a Wiremock server up for all tests

Reply:
This is a good point indead ;)

Reply:
:)